### PR TITLE
NEW add ORDER_SUPPLIER_APPROUVE2 trigger

### PR DESCRIPTION
**Pull request by FlorentPoinsaut**
# NEW add ORDER_SUPPLIER_APPROUVE2 trigger

### CODE REVIEW

File: htdocs/core/class/notify.class.php [hunk 1]
```diff
@@ -79,6 +79,7 @@ class Notify
 		'FICHINTER_ADD_CONTACT',
 		'ORDER_SUPPLIER_VALIDATE',
 		'ORDER_SUPPLIER_APPROVE',
+		'ORDER_SUPPLIER_APPROVE2',
```
Comment (eldy) [inline htdocs/core/class/notify.class.php:82]: No need to duplicate the trigger code
You can reuse ORDER_SUPPLIER_APPROVE and set 
$this->context['secondlevel'] = 1;
before calling
$result = $this->call_trigger($trigger, $user)

Comment (FlorentPoinsaut) [reply htdocs/core/class/notify.class.php:82]: Hum ok but if I don't add a trigger code, how could I set a dedicated email template?

Comment (eldy) [reply htdocs/core/class/notify.class.php:82]: > Hum ok but if I don't add a trigger code, how could I set a dedicated email template? 

With tha var into the context that is different

Comment (FlorentPoinsaut) [reply htdocs/core/class/notify.class.php:82]: Oh I see, context was introduced after the release v16.
I put this PR in draft and I update when my client upgrade to the v18.

**Pull request closed without merging**